docs/users/basics/transactions.md

```diff
@@ -36,7 +36,53 @@ Explain that transactions can interoperate with other blockchains.
 
 ### Ethereum Transactions
 
-<!-- TODO: transactions that interact with the EVM -->
+Evmos supports the following Ethereum transactions.
+
+#### Legacy Transactions
```

Comment:
we should add a note that unprotected legacy transactions are not supported by default